test/torch_optimizer_test.py

```diff
@@ -516,29 +534,26 @@ def test_dynamic_win_put_optimizer(device, kwargs):
         test_mse[-3:].max() < error_threshold*problem_builder.noise_level**2
     ), "Train MSE in the last three epochs doesn't coverge."
 
+
 local_aggregation_scenarios = []
 local_aggregation_scenarios.append(
     pytest.param("CPU", bf.CommunicationType.empty, {"ATC": False, "error_threshold": 2},
                  id="AWC Empty on CPU"))
-# TODO(ybc): Support local aggregation scenario for ATC optimizers
 local_aggregation_scenarios.append(
     pytest.param("CPU", bf.CommunicationType.empty, {"ATC": True, "error_threshold": 2},
-                 id="ATC Empty on CPU",
-                 marks=pytest.mark.skip(reason="ATC doesn't support local aggregation yet")))
+                 id="ATC Empty on CPU"))
 local_aggregation_scenarios.append(
     pytest.param("CPU", bf.CommunicationType.allreduce, {"ATC": False},
                  id="AWC Allreduce on CPU"))
 local_aggregation_scenarios.append(
     pytest.param("CPU", bf.CommunicationType.allreduce, {"ATC": True},
-                 id="ATC Allreduce on CPU",
-                 marks=pytest.mark.skip(reason="ATC doesn't support local aggregation yet")))
+                 id="ATC Allreduce on CPU"))
 local_aggregation_scenarios.append(
     pytest.param("CPU", bf.CommunicationType.neighbor_allreduce, {"ATC": False},
                  id="AWC Neighbor Allreduce on CPU"))
 local_aggregation_scenarios.append(
-    pytest.param("CPU", bf.CommunicationType.neighbor_allreduce, {"ATC": True},
-                 id="ATC Neighbor Allreduce on CPU",
-                 marks=pytest.mark.skip(reason="ATC doesn't support local aggregation yet")))
+    pytest.param("CPU", bf.CommunicationType.neighbor_allreduce,
+                 {"ATC": True, "error_threshold": 10}, id="ATC Neighbor Allreduce on CPU"))
```

Comment:
Why converge slowly under this scenario?

Reply:
Aha, forgot to change it back. Because at the beginning, ACT is not converged, which is strange because AWC converged under the same step size. After diminishing the step-size, it works fine. I changed this back.